api/queries_repo.go

```diff
@@ -914,13 +918,23 @@ func RepoMetadata(client *Client, repo ghrepo.Interface, input RepoMetadataInput
 			return err
 		})
 	}
-	if input.Projects {
+
+	if input.ProjectsV1 {
+		g.Go(func() error {
+			var err error
+			result.Projects, err = v1Projects(client, repo)
```

Comment:
Shouldn't this field (i.e., `RepoMetadataResult.Projects`) be renamed to `ProjectsV1`? I know this might affect multiple places, but it will surely makes things less confusing.

Reply:
Yes, that is a good idea. Will address after stack is merged.